### ISSUE TITLE
Mass delete?

### ISSUE THREAD

**Issue by kmaustral** [joerncodes/obsidian-expiring-notes#1]
This may not be the fault of this plug-in, but I found that all of my files were being deleted until I disabled this plug-in in BRAT. It seems a great idea but may best be used after testing and available on community plug-ins.

**Comment by joerncodes** [comment]
That is strange and obviously not the point of the plugin. Can you give me some background so I can maybe reproduce the error? Did you set the expiry dates yourself or with the command? What operating system are you on? I'd love to fix your problem as soon as possible!

**Comment by kmaustral** [comment]
I hadn't yet set any expiry dates yet. It may not be directly related to
your plug-in. But it seemed to stop when I removed it. I'm on Windows and
also sync to Linux. I noticed something wrong first when I couldn't open
DNP. The folder location in the Daily Notes was red, even though the folder
was still on my system. I couldn't open any Daily Note. Then I checked the
sync and saw a stream of files being deleted.

**Comment by joerncodes** [comment]
Hm, that is indeed strange. In my tests, the plugin didn't touch files without an expiry date.
The collector only finds files that a) have frontmatter in Obsidian and b) have a frontmatter key called `expires` (or whatever you set it to in settings).

Do you think it's possible the files got deleted by something external and were then sync'ed over?

**Comment by kmaustral** [comment]
Yes, I think it is possible. But just that it seemed to stop when I
disabled your plugin. It could have been a coincidence. Hopefully, it's
nothing to do with your plugin. I'll install it again if there's been no
other reports over the next couple of weeks.

**Comment by joerncodes** [comment]
That would indeed be a weird coincidence. I'll do some more tests, and I'll also make sure to ask for more people to beta test in the Discord. Thank you for your input, I hope the error doesn't occur again.
